components/gfx/platform/windows/font.rs

```diff
@@ -40,11 +42,205 @@ impl FontTableMethods for FontTable {
     }
 }
 
+fn make_tag(tag_bytes: &[u8]) -> FontTableTag {
+    assert_eq!(tag_bytes.len(), 4);
+    unsafe { *(tag_bytes.as_ptr() as *const FontTableTag) }
+}
+
+macro_rules! try_lossy(($result:expr) => (try!($result.map_err(|_| (())))));
+
+// Given a set of records, figure out the string indices for the family and face
+// names.  We want name_id 1 and 2, and we need to use platform_id == 1 and
+// language_id == 0 to avoid limitations in the truetype crate.  We *could* just
+// do our own parsing here, and use the offset/length data and pull the values out
+// ourselves.
+fn get_family_face_indices(records: &[truetype::naming_table::Record]) -> Option<(usize, usize)> {
+    let mut family_name_index = None;
+    let mut face_name_index = None;
+
+    for i in 0..records.len() {
+        // the truetype crate can only decode mac platform format names
```

Comment:
Probably? I actually don't think we ever use the family and face name from downloaded fonts, since I believe you specify those in the `@font-face` anyway.  If someone can confirm that, we can change this around to just give dummy UUIDs or something for family/face, and then the only thing left would be the `OS/2` table, which is very simple to parse... in which case we could just do everything ourselves.